impexp-kml-collada-plugin/src/main/java/org/citydb/modules/kml/database/KmlExporterManager.java

```diff
@@ -650,23 +663,137 @@ public void print(ColladaBundle colladaBundle, long id, boolean balloonInSeparat
 		}
 	}
 
+	// set paths to images to the same folder of tiles
+	private void setInitFromOfColladaImages(ColladaBundle colladaBundle, CityGMLClass cityGMLClass, String implicitId) {
+		List colladaObj = colladaBundle.getCollada().getLibraryAnimationsOrLibraryAnimationClipsOrLibraryCameras();
+		if (colladaObj.size() > 0) {
+			for (Object obj : colladaObj) {
+				if (obj instanceof LibraryImages) {
+					List<Image> collada_images = ((LibraryImages) obj).getImage();
+					for (Image colladaImage : collada_images) {
+						colladaImage.setInitFrom(getTextureExportLocation(null, implicitId) + File.separator + cityGMLClass + "_" + colladaImage.getInitFrom());
+					}
+					break;
+				}
+			}
+		}
+	}
+
 	private void convertColladaToglTF(ColladaBundle colladaBundle, File buildingDirectory, File colladaModelFile, File gltfModelFile, boolean exportGltfV1) {
+		convertColladaToglTF(colladaBundle, buildingDirectory, null, colladaModelFile, gltfModelFile, exportGltfV1);
+	}
+
+	private void convertColladaToglTF(ColladaBundle colladaBundle, File buildingDirectory, String implicitId, File colladaModelFile, File gltfModelFile, boolean exportGltfV1) {
 		String collada2gltfPath = config.getProject().getKmlExporter().getPathOfGltfConverter();
 		File collada2gltfFile = new File(ClientConstants.IMPEXP_HOME.resolve(collada2gltfPath).toString());
 		if (collada2gltfFile.exists()) {
-			ProcessBuilder pb = new ProcessBuilder(collada2gltfFile.getAbsolutePath(), "-i", colladaBundle.getGmlId() + ".dae", "-o", gltfModelFile.getAbsolutePath(), "-v", exportGltfV1 ? "1.0" : "2.0");
+			List<String> commands = new ArrayList<>();
+			commands.add(collada2gltfFile.getAbsolutePath());
+			commands.add("-i");
+			commands.add(colladaModelFile.getAbsolutePath());
+			commands.add("-o");
+			commands.add(gltfModelFile.getAbsolutePath());
+			commands.add("-v");
+			commands.add(exportGltfV1 ? "1.0" : "2.0");
+			if (!config.getProject().getKmlExporter().isEmbedTexturesInGltfFiles()) {
```

Comment:
Moved to branch [glb-draco-support](https://github.com/3dcitydb/importer-exporter/tree/glb-draco-support).